service/resource/endpoint/resource_test.go

```diff
@@ -1 +1,6 @@
 package endpoint
+
+import "testing"
+
+func Test_Resource_Endpoint_constructK8sEndpoint(t *testing.T) {
```

Comment:
I want to test this function here, as it is quite complicated and crucial.

Reply:
So when will this happen?